pkg/controller/util/placement.go

```diff
@@ -51,21 +57,21 @@ func GetPlacementDirective(rawPlacement *unstructured.Unstructured) (*PlacementD
 	if err != nil {
 		return nil, err
 	}
-	selector, err := metav1.LabelSelectorAsSelector(placement.Spec.ClusterSelector)
+	selector, err := metav1.LabelSelectorAsSelector(placement.Spec.Placement.ClusterSelector)
```

Comment:
s/placement.spec/resource.spec/  as it becomes confusing percolating from comment on `line 90` in `placement.go`

Reply:
I've changed `rawResource` to `resource` in this method, but I think `placement` is the appropriate name here since it is an instance of `GenericPlacement`.  

Also, I envision a future iteration that unmarshals an unstructured federated type to an internal api struct rather than marshaling separately for overrides and placement, which should simplify things.

Reply:
Ok